src/crypto/tls/common.go

```diff
@@ -469,6 +479,20 @@ const (
 	ECDSAWithSHA1 SignatureScheme = 0x0203
 )
 
+// A TLSFlag is the index of the bit in the TLSFlags bit array that should be
+// set to send the flag. Because TLSFlags can have so many different uses, many
+// yet to be defined, this extension should not be copied into the ECH
+// ClientHelloOuter
+type TLSFlag uint16
+
+const (
+	// ExperimentalFlagSupportMTLS is a flag that signals that a client
+	// supports mTLS, and will be able to respond to CertificateRequest
+	// messages appropriately.
```

Comment:
Would it be more correct to say that the flag "prompts" the server to send a CertificateRequest?

Reply:
Technically no, because the server MAY ignore it, even if it supports it. DavidBen persuaded me to keep it phrased like this so that it was always clear what to do. It does mean that req mTLS is possibly not a great name any more.